src/main/java/com/hazelcast/commandline/HazelcastCommandLine.java

```diff
@@ -0,0 +1,99 @@
+package com.hazelcast.commandline;
+
+import com.hazelcast.commandline.member.MemberCommandLine;
+import picocli.CommandLine;
+
+import java.io.PrintStream;
+import java.util.List;
+import java.util.concurrent.Callable;
+
+import static com.hazelcast.instance.BuildInfoProvider.getBuildInfo;
+import static picocli.CommandLine.*;
+
+@Command(
+        name = "hazelcast",
+        description = "Utility for the Hazelcast IMDG operations." +
+                "%n%n" +
+                "Global options are:%n",
+        versionProvider = HazelcastVersionProvider.class,
+        mixinStandardHelpOptions = true,
+        sortOptions = false
+)
+public class HazelcastCommandLine implements Callable<Void> {
+
+    public static String HAZELCAST_HOME = System.getProperty("user.home") + "/.hazelcast";
+    public final static String SEPARATOR = System.getProperty("file.separator");
+
+    @Mixin(name = "verbosity")
+    private Verbosity verbosity;
+
+    public Void call() {
```

Comment:
Sorry for ignorance again, but what is this method for?

Reply:
picocli needs `Callable` or `Runnable` implementations for commands. Because of that, we need to implement `Callable` interface.

Reply:
Thanks for explanation!